### PR TITLE
Peli-Kiera: Fix alignment issues & missing updated README.md

### PR DESCRIPTION
**Pull request by aleylara**

### CODE REVIEW

File: Peli-Kiera/README.md [hunk 1]
```diff
@@ -78,17 +101,18 @@ from __future__ import unicode_literals
 AUTHOR = 'Author'
 COPYRIGHT = '2019'
 SITENAME = 'Site-Name'
-SITEURL = ''
-SITESUBTITLE = Site Subtitle'
+SITEURL = 'https://my-pelican-blog-address.io'
```
Comment (justinmayer) [inline Peli-Kiera/README.md:104]: Might I suggest using https://www.example.com here instead? Browsers will at least render that properly, and it avoids potential situations where malware could be (eventually) hosted on this currently-fictitious domain. 😊

File: Peli-Kiera/README.md [hunk 2]
```diff
@@ -133,7 +157,7 @@ The pagination is presented with the current page and 3 neighboring pages on eit
       
 Images can be moved on page by adding the tag `#float-right`, `#float-left`, `#mid` and `#full` in your markdown.
 
-```markdown
+```bash
 ![Title]({static}/images/my_image-600x400.png#float-right "")
```
Comment (justinmayer) [inline Peli-Kiera/README.md:161]: Why the change to `bash` here, even though it's Markdown? Just curious.

**Review by justinmayer — COMMENTED** [review]
Nice work, @aleylara. I made two minor comments. Let me know what you think?

File: Peli-Kiera/README.md (hunk 2 above)
Comment (aleylara) [reply Peli-Kiera/README.md]: Hi @justinmayer ,

I had issues with that line on GitHub as well but it seems fine now. 
Short answer: it was just a work around and I am not sure on how to address it on my `pygment.css`

On my local machine I was getting this. 
image

I got something similar for the JSON syntax highlighting as well yesterday. 
So I decided to remove the error class highlighting from `pygment.css`. That solved it. 

I appreciate the feedback and any other recommendations you may still have. Btw does it require a new PR ??

File: Peli-Kiera/README.md (hunk 1 above)
Comment (aleylara) [reply Peli-Kiera/README.md]: Great call.. Thank you!

**Comment by aleylara** [comment]
Hi @justinmayer ,
I think I am done with this PR.
I have no further changes to be made unless you have any other suggestions.

**Comment by justinmayer** [comment]
Looks good. Nice work! 🥇
